docs_versioned_docs/version-ros2humble/ros/config/yaml/platform.mdx

```diff
@@ -0,0 +1,290 @@
+---
+title: Platform
+sidebar_label: Platform
+sidebar_position: 4
+toc_min_heading_level: 2
+toc_max_heading_level: 5
+---
+Every robot platform has unique structures such as versatile sensor mounting solutions, wireless charging receivers, and waterproofing enclosures; we refer to these as attachments.
+
+## Joystick Controller
+We support two types of controllers:
+  - **ps4**: standard Playstation4 controller.
+  - **logitech**: Logitech F710
+
+```yaml
+controller: ps4 # or logitech
+```
+
+## Attachments
+Each platform has its own unique set of attachments. To facilitate customizability, any attachment from any platform can be added on any other platform.
+
+There is a default `parent` links at which these attachments are added; however, the `parent` link can be change to wherever you'd want the attachment to be placed.
+
+Additionally, all attachments can be moved around using the `xyz` parameter to translate and the `rpy` parameter to rotate.
+
+Each attachment requires two parameters. The `name` parameter that will define the name of the links and joints added to URDF, and the `type` parameter which defines which type of attachment is to be added.
+
+Some attachment types will have the same `type` parameter across different platforms. In that case, if you'd like to use the attachment that belongs to a specific platform, you will need to prefix the `type` parameter with the platform that it belongs to. For example, `top_plate` would become `a200.top_plate`. If no prefix is set, then the platform defined in the `serial_number` will be used.
+
+### Husky A200
+import A200Bumper from "/docs_versioned_docs/version-ros2humble/components/yaml/attachments/a200/bumper.mdx";
```

Comment:
Move imports to the top of the MDX file.

Reply:
Done